match_merger.py

```diff
@@ -11,22 +12,42 @@ class MatchMerger(Executor):
     The MatchMerger merges the results of shards by appending all matches..
     """
 
-    def __init__(self, default_traversal_paths: Tuple[str, ...] = ('r',), **kwargs):
+    def __init__(
+        self,
+        default_traversal_paths: Tuple[str, ...] = ('r',),
+        metric_name: str = 'cosine',
```

Comment:
```suggestion
        metric: str = 'cosine',
```

To keep this consistent with argument name in `DocumentArray.match`